### ISSUE TITLE
Loki test passes even if there is differences between two images

### ISSUE THREAD

**Issue by Geloloboy** [oblador/loki#214]
I've started getting this problem after moving from v0.18.1 to v0.19.0 (and anything above it).

I haven't changed anything else and it was working well at v0.18.1

Below is the test command I run:
`loki test --chromeFlags='--headless --no-sandbox --disable-gpu --hide-scrollbars' --chromeRetries 3 --chromeTolerance 2.3 --requireReference --diffingEngine looks-same --chromeDockerImage yukinying/chrome-headless-browser:81.0.4000.3`

I've tried moving to v0.19.1/v0.20.0, using the new fileNameFormatter config, switching between gm/looks-same, but the problem still stands

**Comment by oblador** [comment]
Tolerance of `2.3` is fairly high, could you try decreasing it?

**Comment by Geloloboy** [comment]
I removed the snippet `--chromeTolerance 2.3` but it still passed the test

**Comment by oblador** [comment]
Would you be able to add a test in the example in this repo to reproduce it, so I can have a look?

**Comment by Geloloboy** [comment]
I was able to replicate it on the `react` example in this repo.

What I basically did was I changed the reference file of one of the stories in each of the config to something else and ran the test.

i.e. I modified the following files:
chrome_a4_Long_Element_default.png
chrome_iphone7_Long_Element_default.png
chrome_laptop_Long_Element_default.png

with:
chrome_laptop_Asynchronous_render_Logo_without_delay.png

The full steps are:
1. clone this repo
2. navigated to the example/react folder
3. npm install
4. npm run storybook
5. npm start
6. Modified the files I described above
7. npm test

Do note that I modified the link in the `ShrikhandFont.css` file from `https:` to `http:` since I'm getting request failed to load errors when the loki run starts
